main.go

```diff
@@ -118,15 +118,26 @@ func destroyDNS(c *conf, hc *http.Client) error {
 	req.Header.Add("Accept", "application/json")
 	req.Header.Add("Authorization", fmt.Sprintf("Basic %s", c.konsAPIKey))
 
-	_, err = hc.Do(req)
+	response, err := hc.Do(req)
 	if err != nil {
 		return err
 	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		// if status is not 200, log it, but do not consider it as a service failure
```

Comment:
See Edit in my comment for a suggested improvement.

We have asked the tooling team, the response was that it was too complicated to implement and to use the delete/create method.